internal/store/postgres/migrations/000009_add_grant_source_column.down.sql

```diff
@@ -0,0 +1,18 @@
+BEGIN;
+
+ALTER TABLE
+  "grants"
+ADD
+  COLUMN IF NOT EXISTS "created_by" text;
+
+UPDATE
+  "grants"
+SET
+  "created_by" = "owner";
```

Comment:
are we renaming column here? or set the column `created_by` value for all rows to  "`owner`".

Reply:
Yes, we are renaming column `created_by` to `owner`, and in this `down` SQL script, setting the value of `created_by` to `"grants"."owner"`